package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Bookmarklet for quickly detecting technologies in use on the web",
   "scripts": {
     "build": "babel tech-audit.js --out-file docs/tech-audit.bookmarklet.js",
-    "lint": "eslint tech-audit.js",
-    "lint-and-fixup": "eslint --fix tech-audit.js",
+    "lint": "./node_modules/.bin/eslint tech-audit.js",
+    "lint-and-fixup": "./node_modules/.bin/eslint --fix tech-audit.js",
```

Comment:
These changes shouldn't be necessary; the PATH is prefixed with `node_modules/bin` for these scripts